### PR TITLE
Review w2 pr1

### PR DESCRIPTION
**Pull request by Yorkio**
1. Best Time to Buy and Sell Stock (Easy)

### CODE REVIEW

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Solution {
+    const int INF = 1000 * 1000 * 1000;
```
Comment (andriibugaiov) [inline Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:2]: INT_MAX?

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    
+public:
+    int maxProfit(vector<int>& prices) {
+        // We move from the beginning to the end with one for loop.
+        // At each iteration we try to greedily update the answer.
```
Comment (andriibugaiov) [inline Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:7]: Next time, please, include details on how you would do it. Too abstract comment.

**Review by andriibugaiov — APPROVED** [review]

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp (hunk 1 above)
Comment (Yorkio) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:2]: We can also use INT_MAX.
Bad thing is that in ms compiler we will need to make new include (#inclued <climits>) if we want to use this constant.

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp (hunk 2 above)
Comment (Yorkio) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:7]: I'll try.

Comment (Yorkio) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:7]: Can we merge it?

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp (hunk 1 above)
Comment (andriibugaiov) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:2]: It is ok to include. The code is clean then.

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp (hunk 2 above)
Comment (andriibugaiov) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:7]: Yea, I only approve changes. You merge them.

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp (hunk 1 above)
Comment (Yorkio) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:2]: Why it is unclear with INF constant?

File: Week2/121_Best_Time_To_buy_Sell_Stocks.cpp (hunk 2 above)
Comment (Yorkio) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:7]: Ok, so can I merge it?

Comment (andriibugaiov) [reply Week2/121_Best_Time_To_buy_Sell_Stocks.cpp:7]: Well, you need to look to the state of the review below. It says, "Changes approved".

**Pull request merged**